server/api/impl/admin/provision.go

```diff
@@ -36,6 +36,7 @@ func (adm *Admin) ProvisionApp(cc *apps.Context, sessionToken apps.SessionToken,
 
 	app := &apps.App{
 		Manifest:       manifest,
+		Status:         apps.AppStatusInstalled,
```

Comment:
Why do we set the `AppStatusInstalled` on the `ProvisionApp` call (as opposed to the `InstallApp` call)?

Reply:
Good point. Maybe I've misunderstood what `ProvisionApp` does. Let me double check and come back to you.